src/main/java/org/spongepowered/common/registry/builtin/supplier/SpawnTypeSupplier.java

```diff
@@ -0,0 +1,39 @@
+package org.spongepowered.common.registry.builtin.supplier;
+
+import org.spongepowered.api.CatalogKey;
+import org.spongepowered.api.event.cause.entity.spawn.SpawnType;
+import org.spongepowered.common.data.type.SpongeSpawnType;
+
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+public class SpawnTypeSupplier {
```

Comment:
Same here, stream package and named like the stream classes.

In all honesty, the supplier classes are simply for supplying Vanilla registry entries.